src/main/java/io/kontur/disasterninja/controller/FeaturesController.java

```diff
@@ -26,15 +28,31 @@ public class FeaturesController {
 
     public static final String PATH = "/features";
     private final UserProfileClient userProfileClient;
+    private final LiveSensorFeatureService liveSensorFeatureService;
 
     @Operation(tags = "Features", summary =
             "Get features for app id allowed for user by username retrieved from token "
                     + "(including public ones)", deprecated = true)
     @ApiResponse(responseCode = "200", content = @Content(mediaType = APPLICATION_JSON_VALUE,
             array = @ArraySchema(schema = @Schema(implementation = FeatureDto.class))))
     @GetMapping
+    @Deprecated
     public List<FeatureDto> getUserAppFeatures(
             @RequestParam(name = "appId") @Parameter(name = "appId") UUID appId) {
         return userProfileClient.getUserAppFeatures(appId);
     }
+
+    @Operation(tags = "Features", summary = "Append user's live sensor data")
```

Comment:
Can it be not live sensor data but just generic appending feature to the layer, with the rest of it handled on FE side?

Reply:
I don't think so. This has quite specific logic handling userId and using default disaster.ninja user to append features into layer.
Layers-API would not let anyone to modify layers. Only layer owners can do. I've created live-sensor layer with disaster.ninja owner.

Reply:
It looks then that we need to backlog a ticket to redo the access model on the Layers API. Definitely not only the owner can edit the layers, but also people the owner opened some capabilities to. This particular use case shows that we have at least owner, logged-in, anonymous users kind and at least read, write, append capabilities. I believe that stamping the user into the feature can be done in a nicer way, like having a special data type for certain field in layer descriptor and just always stamping it on write if the spot to stamp the user at is present at the layer write.